src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -1,11 +1,7 @@
 package seedu.addressbook.data.person;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.Iterator;
-import java.util.List;
+import java.util.*;
```

Comment:
The coding standard for CS2103 specifies to import just the specific classes you need. If this happens automatically, you can change "code style -> Java -> imports" and change the setting